build.gradle

```diff
@@ -140,22 +147,10 @@ task buildImage {
     dependsOn tasks.docker
 }
 
-sourceSets {
-    integrationTest {
-        java {
-            compileClasspath += main.output + test.output
-            runtimeClasspath += main.output + test.output
-            srcDir 'src/itest/java'
-        }
-        resources.srcDir 'src/itest/resources'
-    }
-}
-
 task itest(type: Test) {
     group 'Verification'
     description 'Runs the integration tests.'
-    dependsOn   buildImage
-    setTestClassesDirs(sourceSets.integrationTest.output)
-    classpath = sourceSets.integrationTest.runtimeClasspath
-    useJUnitPlatform()
```

Comment:
Does `itest` still use JUnit 5 without this line?

Reply:
The shared configuration is:
* we use JUnit5
* a coverage report will be generated after

Reply:
Ok I see.
But in that case do we need to run test coverage after integration tests?

Reply:
As it is implemented, there is no harm done.
When applying the **jacoco** plugin, all test tasks are instrumented by default to produce coverage data.
Here, we only configure the system to generate **coverage reports** after all test tasks have been executed.

Reply:
Ok no problem. We can keep it that way.